src/localization/authStrings.json

```diff
@@ -39,6 +39,17 @@
     "password": "ລະຫັດຜ່ານ",
     "user_name": "ຊື່​ຜູ້​ນຳ​ໃຊ້"
   },
+  "my": {
+    "logging_in": "Login ဝင်ရောက်နေသည်",
```

Comment:
actually, the `Login` here is redundant. Google translate has `ဝင်ရောက်နေသည်` == `Signing in`